rules/data_binding.bzl

```diff
@@ -254,11 +250,11 @@ def _process(
     if defines_resources:
         # Outputs of the Data Binding annotation processor.
         br_out = ctx.actions.declare_file(
-            output_dir + "bin-files/%s-br.bin" % java_package,
+            output_dir + "bin-files/%s--br.bin" % java_package,
```

Comment:
Why the double dash?

Reply:
I'm not entirely sure, but this is necessary on the version of android_tools currently pulled. (Presumably output by the data binding compiler.) If I override android_tools with a new version (e.g. Grab has one [here](https://github.com/grab/grab-bazel-common/tree/master/android/tools)), then I no longer need the double dash.